spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
By doing this there are lots of duplication code in your tests.

And that's the meaning of nesting,  I don't think we can gain much from
running tests in parallel.

On Tue, Jan 27, 2015 at 3:41 PM, Joel Low <notifications@github.com> wrote:

> In spec/features/global_announcements_spec.rb
> <https://github.com/Coursemology/coursemology2/pull/78#discussion_r23591384>
> :
>
> > @@ -0,0 +1,57 @@
> > +require 'rails_helper'
> > +
> > +RSpec.describe 'Global announcements', type: :feature do
> > +  subject { page }
> > +  let(:instance) { create(:instance) }
> > +
> > +  with_tenant(:instance) do
> > +    describe 'no global announcements' do
> > +      before do
> > +        instance.announcements.clear
>
> By doing that you cannot run your tests in parallel.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/78/files#r23591384>.
>